src/coreclr/src/jit/compiler.h

```diff
@@ -6364,6 +6364,11 @@ class Compiler
     };
 
     static const size_t s_optCSEhashSize;
+    static const size_t s_optCSEhashGrowthFactor;
```

Comment:
This could be renamed to indicate that it is the initial hash table size

s_optCSEhashSize  => s_optCSEhashSizeInitial